src/lwc/rd2ElevateInformation/rd2ElevateInformation.js

```diff
@@ -84,6 +84,8 @@ export default class rd2ElevateInformation extends NavigationMixin(LightningElem
         alert: ''
     };
     @track error = {};
+    @track elevateCommitmentURL;
```

Comment:
Let's use a getter method instead of the var to get the URL.

Reply:
Oops.  Missed this one.  It is still outstanding.